src/app.ts

```diff
@@ -79,7 +81,7 @@ client.on('message', msg => {
 
 	// 100% not an alias, so it must be a command class.
 	try {
-		new (command as Constructable<BaseCommand>)(msg).execute();
+		new (command as Constructable<BaseCommand>)(msg, services).execute();
```

Comment:
```suggestion
		new (command as Constructable<BaseCommand>)(msg).execute();
```
Correction to what we were talking about, we don't really need to pass a services object if we just plan to create one in `base-command.ts`